arches/app/utils/index_database.py

```diff
@@ -327,30 +327,41 @@ def index_resources_by_type(
 
     for resource_type in resource_types:
         start = datetime.now()
+        try:
+            uuid.UUID(str(resource_type))
+            graph = models.GraphModel.objects.get(graphid=str(resource_type))
+        except ValueError:
+            try:
+                graph = models.GraphModel.objects.get(slug=str(resource_type))
+            except:
+                logger.warning(
+                    "Unable to resolve resource type %s. Please confirm it is a valid graph ID or slug."
+                    % resource_type
+                )
+                continue
```

Comment:
Is there a reason to have the log & continue only under one failure path? A UUID not in the db will still hard-crash.